play-json/shared/src/test/scala/play/api/libs/json/JsResultSpec.scala

```diff
@@ -48,4 +48,37 @@ class JsResultSpec extends WordSpec with MustMatchers {
       JsError("err").recoverWith { _ => succ } mustEqual succ
     }
   }
+
+  "JsResult" should {
+
+    "return true for JsSuccess(x)#contains(x)" in {
+      val x = 10
```

Comment:
Do not need such trivial `val`

Reply:
@cchantep I don't agree with you. I think consistent test values and absence of magic numbers is good for tests.
If you would like to, I can rewrite tests with table driven cases.
Moreover, you can do it yourself :)

Reply:
There I see no value to declared such expected result as `val` as it's pure/simple/literal.

Reply:
Do you think it is better to have
`assert(JsSuccess(10).contains(10))`?